src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/sessions/UpdateSessionTemplateDto.kt

```diff
@@ -40,9 +40,6 @@ data class UpdateSessionTemplateDto(
   @Schema(description = "biWeekly time table", example = "true", required = true)
   val biWeekly: Boolean? = null,
 
-  @Schema(description = "list of included prisoner categories", required = false)
-  val includedPrisonerCategories: List<String>? = null,
-
-  @Schema(description = "list of excluded prisoner categories", required = false)
-  val excludedPrisonerCategories: List<String>? = null,
+  @Schema(description = "list of group references for allowed prisoner category groups", required = false)
+  val categoryGroupReferences: List<String>? = listOf(),
```

Comment:
Should this be PrisonerCategory

Reply:
Based this on the locationGroupReferences in the same class. I thought it takes the group references as a list.

Reply:
yes I was being stupid